conf/country-codes-en.properties

```diff
@@ -0,0 +1,249 @@
+AFG=Afghanistan
```

Comment:
Hmm, if we now what are the codes, maybe it's better to encode it scala?

Reply:
Done